src/v2/Apps/Auction/Routes/ConfirmBid/index.tsx

```diff
@@ -32,13 +32,20 @@ import {
   FormValuesForBidding,
   createStripeWrapper,
   determineDisplayRequirements,
+  errorMessageForBidding,
   errorMessageForCard,
+  saleConfirmRegistrationPath,
   toStripeAddress,
 } from "v2/Apps/Auction/Components/Form"
 
 const logger = createLogger("Apps/Auction/Routes/ConfirmBid")
 
 type BidFormActions = FormikActions<FormValuesForBidding>
+// TODO: Replace with a GraphQL type
+interface BidderPosition {
```

Comment:
Is it possible to use a part of `ConfirmBidCreateBidderPositionMutationResponse` instead here? Maybe a `Partial<ConfirmBidCreateBidderPositionMutationResponse["createBidderPosition"]>`? Seems like this isn't actually a bidder position, but more the metadata about the bidder position request response - maybe `BidderPositionResponseMetadata`?

More food for thought – just want to see if it's possible to re-use some types here!

Reply:
`Partial` makes all properties optional while we want `status` and `messageHeader` to exist. What we need here is an `Intersection` type for the types below that specifically requires `status` and `messageHeader`, and I just wasn't sure how we could do this (we handle two different types of GraphQL response here, so it's sadly more than just re-using a type).

 * `ConfirmBidCreateBidderPositionMutationResponse["createBidderPosition"]` and
 * `BidderPositionQueryResponse["data"]["me"]["bidderPosition"]["position"]`